### PR TITLE
Forms Automated Test - LPS-132262 | master

### PR DESCRIPTION
**Pull request by chaowang-123**
https://issues.liferay.com/browse/LPS-132262

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase [hunk 1]
```diff
@@ -43,13 +43,38 @@ definition {
...
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Multiple Selection");
+
```
Comment (brittneyq) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase:62]: Hi @chaowang-123, can you please add step 5 in the ticket, "Assert the multiple selection field is no longer present" in your test? Thank you!

**Comment by chaowang-123** [comment]
ci:test:sf

**Comment by chaowang-123** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase (hunk 1 above)
Comment (chaowang-123) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase:62]: Hi @brittneyq , have added, thank you~

**Comment by brittneyq** [comment]
Combined and sent in [39](https://github.com/brittneyq/liferay-portal/pull/39)

**Pull request closed without merging**